perun-cli/getAttribute

```diff
@@ -127,8 +167,15 @@ unless (@attributes) {
 my $table = Text::ASCIITable->new( { reportErrors => 0, utf8 => 0 } );
 $table->setCols( 'ID', 'Name', 'Type', 'Value' );
 
+my $aa=0;
 foreach my $attribute (sort $sortingFunction @attributes) {
-	$table->addRow( $attribute->getId, $attribute->getName, $attribute->getType, $attribute->getValueAsScalar );
```

Comment:
You have conflict on this row, please rebase your commit on current master. You are now missing ``$attribute->isUniqueToPrint`` in table print, which causes this conflict.